src/plugins/ceramic-feed/server/content-types/ceramic-post/schema.js

```diff
@@ -29,17 +29,32 @@ module.exports = {
     picture: {
       type: 'media',
       multiple: false,
+      required: false,
+      configurable: false,
+      allowedTypes: ['images'],
+    },
+    sponsoredLink: {
```

Comment:
should be a separate content type 'sponsor'. please check task description
in the post schema you need to define relation field `sponsored`